edgelet/iotedge/src/init/import/old_config/mod.rs

```diff
@@ -37,14 +37,14 @@ pub(crate) struct Config {
 
 #[derive(Debug, serde_derive::Deserialize)]
 pub(crate) struct Connect {
-    pub(crate) workload_uri: Url,
     pub(crate) management_uri: Url,
+    pub(crate) workload_uri: Url,
```

Comment:
nit: any reason you shuffled around the order in the struct?
not a blocker, just curious

Reply:
It was, and still is, driving me crazy.

The usual order we think of is management -> workload, and the default config file has management -> workload, but the struct in edgelet-core has workload -> management, and the code that uses the settings in iotedged does workload -> management.  It didn't matter until `iotedge init import` arrived and started writing out configs, which is why all the output `edged.yaml` in the test cases have workload -> management. But because the default config file has management -> workload I wanted to keep the input config files of the test cases management -> workload.

With this PR, I added a function that returns two URIs (the defaults), so I had to pick what order it returns them in. The code that uses the defaults would look weird if it parsed the old config in one order and overrode with the defaults in the opposite order. So I just made everything related to the old config in the management -> workload order.